LPPC/Plugins/topsky/GRpluginSettings.txt

```diff
@@ -72,25 +129,72 @@ List_Departure_EOBT=1
 List_Departure_STAND=0
 
 Window_APP_Scale=17
-Window_APP_AltFilter=5000
+Window_APP_AltFilter=24000
 
 [LPPT]
 Airport_Elevation=374
 Airport_Radius=6.5
 Airport_Runway_End=21:N038.47.50.400:W009.07.38.500
 Airport_Runway_End=03:N038.45.56.460:W009.08.39.480
+Airport_Runway_End=35:N008.47.50.400:W089.07.38.500
+Airport_Runway_End=17:N008.45.56.460:W089.08.39.480
+
+List_Inbound=21,0,1060,565
+List_Inbound=03,0,1060,565
+List_Arrival=1,1245,565
+List_Alerts=1,1351,25
+List_Stand=0,1102,25
+List_Stand_Items=20
+
+Window_APP=1,1430,500,490,457
+Window_APP_Extensions=0
+Window_APP_Scale=9
 
 [LPPR]
 Airport_Elevation=227
 Airport_Radius=6.5
 Airport_Runway_End=17:N041.15.47.900:W008.41.06.900
 Airport_Runway_End=35:N041.13.57.200:W008.40.37.900
 
+List_Inbound=17,0,1734,650
+List_Inbound=35,0,1734,650
+List_Arrival=1,1549,650
+List_Alerts=1,1109,25
+List_Stand=0,1102,25
+List_Stand_Items=20
+
+Window_APP=1,700,25,410,410
+Window_APP_Extensions=0
+Window_APP_Scale=14
+
 [LPFR]
 Airport_Elevation=24
 Airport_Radius=6.5
 Airport_Runway_Buffer_LVP=28:160.0
 
+List_Inbound=10,0,1734,650
+List_Inbound=28,0,1734,650
+List_Arrival=1,1549,650
+List_Alerts=1,1109,25
+List_Stand=0,1102,25
+List_Stand_Items=20
+
+Label=GND:UNC:0:COMM,0,0
```

Comment:
this will enable to manually correlate standby tracks, of course that implies the controller needs an alternate way to figure out who is who but that is entirely up to their imagination